types/chrome-apps/index.d.ts

```diff
@@ -6351,11 +6336,10 @@ declare namespace chrome {
      */
     namespace notifications {
         /**
-         * @enum
-         * @prop BASIC - icon, title, message, expandedMessage, up to two buttons.
-         * @prop IMAGE - icon, title, message, expandedMessage, image, up to two buttons.
-         * @prop LIST - icon, title, message, items, up to two buttons. Users on Mac OS X only see the first item.
-         * @prop PROGRESS - icon, title, message, progress, up to two buttons.
+         * BASIC - icon, title, message, expandedMessage, up to two buttons.
+         * IMAGE - icon, title, message, expandedMessage, image, up to two buttons.
+         * LIST - icon, title, message, items, up to two buttons. Users on Mac OS X only see the first item.
+         * PROGRESS - icon, title, message, progress, up to two buttons.
```

Comment:
As a feature request for the autofixer, it'd be nice if these could be moved into JSDoc tags specific to the properties...

Reply:
https://github.com/gajus/eslint-plugin-jsdoc/issues/1054